flip/templates/settings/outcomes.html

```diff
@@ -34,7 +34,14 @@
           {% url 'settings:outcomes_edit' pk=object.pk as edit_url %}
           <tr>
             <td>
-              <a href="{{ edit_url }}">{{ object.title }}</a>
+              <a href="{{ edit_url }}">
+                {{ object.title }}
+                {% if object.doc_type == 'any' %}
+                  (Activity, assessment)
```

Comment:
capfirst Assessment, but to be future proof you should use `TypeOfOutcome.TYPE_CHOICES` here.